src/VisualStudio/CSharp/Impl/CSharpPackage.cs

```diff
@@ -63,7 +63,6 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             try
             {
                 await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
-                await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
```

Comment:
>SwitchToMainThreadAsync [](http://example.com/codeflow?start=42&length=23)

RegisterService doesn't require main thread

Reply:
do we need/want CA(true) on the prior line? if so, doc.

Reply:
no, we don't want that either, I just missed it in this PR. I'll go ahead and change.

Reply:
doc that as well  plz